### PR TITLE
Requires/Wants option in systemd unit file

### PR DESCRIPTION
**Pull request by ovv**

### CODE REVIEW

**Comment by ovv** [comment]
[Doc on systemd unit file](https://www.freedesktop.org/software/systemd/man/systemd.unit.html)

File: roles/system_service/templates/etc/systemd/system/service.j2 [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [Unit]
 Description={{ service_name }}
+{% if service_requires %}Requires={{ service_requires }}
```
Comment (mattrasband) [inline roles/system_service/templates/etc/systemd/system/service.j2]: Looks like you're missing the `{% endif %}` on both of these

**Review by mattrasband — CHANGES_REQUESTED** [review]

File: roles/system_service/templates/etc/systemd/system/service.j2 (hunk 1 above)
Comment (ovv) [reply roles/system_service/templates/etc/systemd/system/service.j2]: Oups

**Comment by ovv** [comment]
@mrasband I think this is ok now ?

**Comment by mattrasband** [comment]
Yep, I think it's good

**Pull request merged**